### DISCUSSION TITLE
Error while using Computation.qmd tutorial document

### DISCUSSION THREAD

**Discussion by bmillerlab** [quarto-dev/quarto-cli#11490] [Q&A]
### Bug description

When running the tutorial there is a problem with purrr after running the chunk:

library(tidyverse)
library(ggplot2)

Error in `purrr::map2_lgl()`:
ℹ In index: 19.
ℹ With name: ragg.
Caused by error in `.make_numeric_version()`:
! invalid non-character version specification 'x' (type: double)
Backtrace:
  1. tidyverse::tidyverse_update()
  2. tidyverse::tidyverse_deps(recursive, repos)
  3. purrr::map2_lgl(cran_version, local_version, `>`)
  4. purrr:::map2_("logical", .x, .y, .f, ..., .progress = .progress)
  8. rlang (local) .f(.x[[i]], .y[[i]], ...)
  9. base::Ops.numeric_version(.x, .y)
 10. base::as.numeric_version(e2)
 11. base::numeric_version(x)
 12. base::.make_numeric_version(x, strict, .standard_regexps()$valid_numeric_version)
 13. base::stop(msg, domain = NA)
Error in purrr::map2_lgl(cran_version, local_version, `>`) : 
ℹ With name: ragg.
Caused by error in `.make_numeric_version()`:
! invalid non-character version specification 'x' (type: double)

### Expected behavior

I continued to the actual steps of the tutorial and so far everything seems to work fine.  However, I am just learning and your site indicatres I should report anything that shows up that "might be a problem."

### Your environment

IDE: Version 2024.09.1+394 (2024.09.1+394)
Model: MacBook Pro 2023
Chip:	Apple M3 Max
MacOS: Sonoma Version 14.6

### Quarto check output

Quarto 1.5.57
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.2.0: OK
      Dart Sass version 1.70.0: OK
      Deno version 1.41.0: OK
      Typst version 0.11.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.5.57
      Path: /Applications/RStudio.app/Contents/Resources/app/quarto/bin

(|) Checking tools....................

**Comment by mcanouil** [comment]
Could you share the exact document you have used, please?
Because the document `computations.qmd` does not have the library you are mentioning in your error message. (https://quarto.org/docs/get-started/computations/rstudio.html)
Note also that the error comes from R, not Quarto CLI, check/update your R packages.
If you execute the code directly in R and this produces the error, then the issue is your setup.

**Comment by cscheid** [comment]
I'm going to convert this to a Q&A, since from the evidence, it's an R question.

**Comment by bmillerlab** [comment]
Hi, Since this is apparently an R problem I'm just going to drop the issue.  I am actually using the most up to date versions of everything though, since I just updated everything yesterday (R, Rstudio) morning before I did this and also updated all my packages.  

I did a little more digging and it seems some other people are having some potentially related purrr issues.  Regardless I actually completed the tutorial.  

It must have been the data visualization tutorial, since apparently it was not the computations tutorial.  I did both yesterday and reported the problem after I was finished.

Thanks,
